x-pack/plugins/observability_solution/infra/public/components/asset_details/hooks/use_dashboards_fetcher.ts

```diff
@@ -6,7 +6,7 @@
  */
 
 import { useState, useEffect } from 'react';
-import type { SearchDashboardsResponse } from '@kbn/dashboard-plugin/public/services/dashboard_content_management/lib/find_dashboards';
+import type { SearchDashboardsResponse } from '@kbn/dashboard-plugin/public/services/dashboard_content_management_service/lib/find_dashboards';
```

Comment:
How about exporting type `SearchDashboardsResponse` from  dashboard/public/index.ts?

Reply:
Good call - done in c8a01ca6d90c20caca3959497a7293f90f5dfb25 :+1: